webpack.config.js

```diff
@@ -131,6 +131,7 @@ const config = {
   resolve: {
     extensions: [".tsx", ".ts", ".js", ".jsx", ".json"],
     modules: [path.resolve(path.join(repo, "./frontend")), "node_modules"],
+    fallback: { path: require.resolve("path-browserify") },
```

Comment:
uh, c'mon

Reply:
Yeah, they stopped including node modules in v5:

```
Module not found: Error: Can't resolve 'path' in '/fleet/node_modules/content-disposition'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.
```